enterprise/frontend/src/embedding-sdk/cli/steps/create-models-and-xrays.ts

```diff
@@ -15,17 +20,19 @@ export const createModelsAndXrays: CliStepMethod = async state => {
   const spinner = ora("Creating models…").start();
 
   try {
+    const models = [];
+
     // Create a model for each table
-    const models = await Promise.all(
-      tables.map(table =>
-        createModelFromTable({
-          table,
-          databaseId,
-          cookie,
-          instanceUrl,
-        }),
-      ),
-    );
+    for (const table of chosenTables) {
+      const model = await createModelFromTable({
+        table,
+        databaseId,
+        cookie,
+        instanceUrl,
+      });
+
+      models.push(model);
+    }
```

Comment:
This has to be sequential, otherwise it causes a race condition where the returned table objects (and thus table IDs) are not deterministic 🤦🏻